android/src/app/organicmaps/MwmApplication.java

```diff
@@ -60,6 +68,16 @@ public class MwmApplication extends Application implements AppBackgroundTracker.
   private final MapManager.StorageCallback mStorageCallbacks = new StorageCallbackImpl();
   private MediaPlayerWrapper mPlayer;
 
+  @NonNull
+  private WeakReference<Activity> mTopActivity = new WeakReference<>(null);
```

Comment:
= null here and in onActivityPaused ?

Then just check for null in getTopActivity

Reply:
Yes, we can do that. Please note that mTopActivity.get() also can return null.